App/Containers/PhotoViewerScreen.js

```diff
@@ -32,55 +50,41 @@ class PhotoViewerScreen extends React.PureComponent {
     )
   }
 
-  get caption () {
-    return (
-      <View style={{ bottom: 0, height: 65, backgroundColor: 'rgba(0, 0, 0, 0.7)', width: '100%', position: 'absolute', justifyContent: 'center' }}>
-        <Text style={{ textAlign: 'center', color: 'white', fontSize: 15, fontStyle: 'italic' }}>{'Some metadata'}</Text>
-      </View>
-    )
-  }
+  // get caption () {
+  //   return (
+  //     <View style={{ bottom: 0, height: 65, backgroundColor: 'rgba(0, 0, 0, 0.7)', width: '100%', position: 'absolute', justifyContent: 'center' }}>
+  //       <Text style={{ textAlign: 'center', color: 'white', fontSize: 15, fontStyle: 'italic' }}>{'Some metadata'}</Text>
+  //     </View>
+  //   )
+  // }
 
   renderImage(image) {
-    const imageData = IPFS.syncGetPhotoData(image.image.hash + '/thumb')
     return (
-      <Image
-        source={{uri: 'data:image/jpeg;base64,' + imageData}}
-        style={image.style}
-        resizeMode={image.resizeMode}
-        capInsets={image.capInsets}
-      />
+      <View style={{ flex: 1, flexDirection: 'row', backgroundColor: 'rgba(0, 0, 0, 1)' }}>
+        <Image
+          source={{uri: 'https://localhost:9080/ipfs/' + image.hash + '/photo'}}
+          style={{ flex: 1, width: null, height: null }}
+          resizeMode="contain"
+        />
+      </View>
     )
   }
 
   render () {
+    const item = this.props.navigation.state.params.item
     return (
       <View style={{flex: 1}}>
         <StatusBar hidden />
-        <Gallery
```

Comment:
Guess I'm a little confused why you removed the `Gallery` component. So now you just just view a single image, but you can't swipe to the next/previous images? And there's probably no zooming either?

Reply:
i didn't think of those other use cases actually :P as per our dicsussion on slack, i'll revert this

Reply:
file has been reverted to a previous state